blueman/services/__init__.py

```diff
@@ -4,12 +4,8 @@
 from __future__ import division
 from __future__ import unicode_literals
 
-from blueman.services.AudioSink import AudioSink
-from blueman.services.AudioSource import AudioSource
 from blueman.services.DialupNetwork import DialupNetwork
 from blueman.services.GroupNetwork import GroupNetwork
-from blueman.services.HandsfreeGateway import HandsfreeGateway
```

Comment:
You're dropping HandsfreeGateway but not considering it for the generic connect. See my comment on the `auto_connect` properties in BlueZ.